src/component/parser/BpmnParser.ts

```diff
@@ -15,12 +15,18 @@
  */
 import BpmnModel from '../../model/bpmn/BpmnModel';
 import BpmnXmlParser from './xml/BpmnXmlParser';
-import BpmnJsonParser from './json/BpmnJsonParser';
+import BpmnJsonParser, { defaultBpmnJsonParser } from './json/BpmnJsonParser';
 
 export default class BpmnParser {
```

Comment:
```suggestion
class BpmnParser {
```

Reply:
it seems it is used only locally - no need to export it

Reply:
Yes, it works
If we remove the export, this won't prevent extension but as we do not support extension for now, it is ok
Do you agree @aibcmars ?

Reply:
yes, good for now